stat/samplemv/sample_test.go

```diff
@@ -94,15 +94,15 @@ func TestRejection(t *testing.T) {
 	dim := 3
 	bounds := make([]distmv.Bound, dim)
 	for i := 0; i < dim; i++ {
-		min := rand.NormFloat64()
-		max := rand.NormFloat64()
+		min := src.NormFloat64()
+		max := src.NormFloat64()
 		if min > max {
 			min, max = max, min
 		}
 		bounds[i].Min = min
 		bounds[i].Max = max
 	}
-	target := distmv.NewUniform(bounds, nil)
+	target := distmv.NewUniform(bounds, src)
```

Comment:
I guess it doesn't hurt for this to be here, though this was intentionally kept as `nil` because no random numbers are generated from `target`.

Reply:
OK. I can revert this if you want; the test still passes without the change (the changes as 97 and 98 were necessary for the test to pass in ways that were difficult to understand - revert those locally to see what happens - you need to run the entire package, not just TestRejection).

I'll wait.

Reply:
Nah, I don't think it needs to be reverted.